OctopusDSC/DSCResources/cOctopusSeqLogger/cOctopusSeqLogger.psm1

```diff
@@ -237,8 +261,8 @@ function Set-TargetResourceInternal {
         $nlogConfig = Get-NLogConfig $nlogConfigFile
 
         #remove then re-add "<add assembly="Seq.Client.NLog"/>" to //nlog/extensions
-        $nlogExtensionElement = ($nlogConfig.nlog.extensions.add | where-object { $_.assembly -eq "Seq.Client.Nlog" })
-        if ($null -ne $nlogPropertyElement) {
+        $nlogExtensionElement = ($nlogConfig.nlog.extensions.add | where-object { $_.assembly -eq "Seq.Client.NLog" })
+        if ($null -ne $nlogExtensionElement) {
```

Comment:
This was a bug that we'd never noticed... It would continually add additional elements here every time it was run, as it wasn't removing the old ones... :facepalm: